tests/integration/test_storage_s3_queue/configs/zookeeper.xml

```diff
@@ -13,19 +13,4 @@
             <port>2181</port>
         </node>
     </zookeeper>
-
-    <remote_servers>
```

Comment:
That was in use in test_storage_s3_queue/test_2.py::test_shards_distributed, let's reintroduce it in remote_servers.xml